libecho/src/IAudioConverter.h

```diff
@@ -10,9 +10,9 @@
 template <typename T>
 class IAudioConverter {
 public:
-    IAudioConverter(QAudioFormat inputFormat, QAudioFormat outputFormat, double windowLength, size_t soundsPerByte)
-        : inputFormat(inputFormat), outputFormat(outputFormat), windowLength(windowLength) {
-        encryptedByteSize = soundsPerByte * outputFormat.sampleRate() * windowLength;
+    IAudioConverter(QAudioFormat inputFormat, QAudioFormat outputFormat, double windowSize, size_t soundsPerByte)
+        : inputFormat(inputFormat), outputFormat(outputFormat), windowSize(windowSize) {
+        encryptedByteSize = soundsPerByte * windowSize;
```

Comment:
Doesn't it break @Riyria-was-taken code?

Reply:
no, windowSize is just sampleRate * windowLength